add sentences about data removal (#440)
AlexsLemonade/OpenPBTA-manuscript: content/06.methods.md

We determined RNA-Seq read strandedness by running the `infer_experiment.py` script from `RNA-SeQC` [@doi:10.1093/bioinformatics/bts196] on the first 200k mapped reads.
We removed any samples whose calculated strandedness did not match strandedness information provided by the sequencing center.
We required that at least 60% of RNA-Seq reads mapped to the human reference for samples to be included in analysis.

> During the OpenPBTA analysis process, if we identified that a sample was mis-identified or potentially swapped, through collaborative analyses and pathology review, we removed it from our data releases and it was taken off of the Kids First portal.

Reviewer: Drop word count (though perhaps this doesn't matter for methods!)

```suggestion
During OpenPBTA analysis, we identified some samples which were mis-identified or potentially swapped.
Through collaborative analyses and pathology review, these samples were removed from our data releases and from the Kids First portal.
```

> Sample removal and reasons were documented in the OpenPBTA-analysis repository [release notes](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/doc/release-notes.md).

Reviewer: ```suggestion
Sample removal and associated justifications were documented in the OpenPBTA data [release notes](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/doc/release-notes.md).
```
